homework/load_csv.py

```diff
@@ -0,0 +1,91 @@
+#Import packages
+import psycopg2
+import pandas as pd 
+from sqlalchemy import create_engine
+
+####################################
+#### Putting data into Postgres ####
+####################################
+
+#Read csv files
+products = pd.read_csv('Desktop/BGSE/term1/data_warehousing/warehousing-operational-databases/homework/extracts/products.csv')
+orders = pd.read_csv('Desktop/BGSE/term1/data_warehousing/warehousing-operational-databases/homework/extracts/orders.csv')
+employees = pd.read_csv('Desktop/BGSE/term1/data_warehousing/warehousing-operational-databases/homework/extracts/employees.csv')
+
+#Create engine to connect to the "extracts" database using Postgres
+engine = create_engine('postgresql://postgres:@localhost:5432/extracts')
+#Create "products" table
+products.to_sql('products', engine, index = False)
+#Create "orders" table
+orders.to_sql('orders', engine, index = False)
+#Create "employees" table
+employees.to_sql('employees', engine, index = False)
+
+##############################
+#### Normalizing the data ####
+##############################
+
+#Connect to the data base using the connector from psycopg2
+conn = psycopg2.connect(database = "extracts", user = "postgres", host = "localhost")
+#CCreate a cursor
+cur = conn.cursor()
+
+#### Products database ####
+
+#Add primary key
+cur.execute(
+	"""
+	ALTER TABLE products 
+		ADD PRIMARY KEY (product_code);
+	"""
+	)
+
+
+#### Employees database
+
+employees_querries = [
+#Add primary key
+"ALTER TABLE employees ADD PRIMARY KEY (employee_number);", 
+#Create new table containing information related to the offices
+"CREATE TABLE offices AS SELECT DISTINCT(office_code), city, state, country, office_location FROM employees;", 
+#Add primary key
+"ALTER TABLE offices ADD PRIMARY KEY (office_code);",
+#Add reference
+"ALTER TABLE employees ADD CONSTRAINT office_code FOREIGN KEY (office_code) REFERENCES offices (office_code);",
+#Drop variables exported to the new table
+"ALTER TABLE employees DROP COLUMN city, DROP COLUMN state, DROP COLUMN country, DROP COLUMN office_location;"]
+
+#Execute querries
+for eq in employees_querries: cur.execute(eq)
+
+
+#### Orders database ####
+
+orders_querries =  [
```

Comment:
Nothing wrong with doing it all from Python, but there's also maybe no benefit? Could be in a .sql file and run directly from a Postgres client. Keeping sql in a .sql file will always be a bit nicer, everyone editing will have syntax highlighting, etc., where as "strings" in python can easily hide bugs!